ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -75,6 +84,11 @@ export default function EditGasDisplay({
             />
           </div>
         )}
+        {showTopError && (
```

Comment:
I'm wondering (thinking out loud) -- could we have a generic `error` prop passed in instead of doing that error logic / display messaging in here?  Or would that be *too* generic, because we could have more than one error?

Reply:
Yeah, I wanted to wait until possible other errors became clear before generalizing